### DISCUSSION TITLE
Can i control Textbox not wrap？

### DISCUSSION THREAD

**Discussion by sanheng** [fabricjs/fabric.js#7232] [Q&A]
In the case of a fixed width, using Textbox to enter a space will wrap.

Set a fixed width, set too much input to automatically reduce the font, when using itext, the border will be too small

**Comment by asturur** [comment]
You can use the changed event on the text object to achieve that.
You can start a codepen/jsfiddle and someone can try to help you.

**Reply by sanheng** [reply]
It will wrap when entering a space, but will not wrap when entering text

https://codesandbox.io/s/textbox-ixoco?file=/src/App.js

**Reply by asturur** [reply]
you can also use the splitByGrapheme property, that has been added for languages that do not use the space to wrap

**Reply by sanheng** [reply]
i fount splitByGrapheme is not work, It will wrap when entering a space

https://codesandbox.io/s/textbox-ixoco?file=/src/App.js

![image](https://user-images.githubusercontent.com/13793256/128691180-8ce52587-1ec1-4c7b-a319-d625516791d9.png)
